packages/cli/src/commands/deploy/index.ts

```diff
@@ -668,11 +668,13 @@ export default async (client: Client): Promise<number> => {
 
     if (err instanceof BuildError) {
       output.error(err.message || 'Build failed');
-      output.error(
-        `Check your logs at https://${now.url}/_logs or run ${getCommandName(
-          `logs ${now.url}`
+      output.log('\n');
+      output.log(
```

Comment:
I think the `error` -> `log` makes sense as long as `log` also goes to `stderr`.
(Ignore if `output.error` goes to `stdout`)